_posts/blog/2015-04-24-data_duplication_writeup.md

```diff
@@ -26,6 +26,7 @@ This issue was introduced into the dataset sometime in May 2014. We fixed the ro
 
 The table below details the effect of the duplication bug on each of the BigQuery tables. Note that the table is organized not by time of query, but according to the time the M-Lab test occurred, as that is how M-Lab data is partitioned into tables in BigQuery.
 
+{:.big-query}
```

Comment:
Comment addressed.  Fixed in https://github.com/m-lab/m-lab.github.io/pull/91/commits/cfd1724768b3c1f78fd772dc279e3cde618b3d6b.